src/Toolkit/Str.php

```diff
@@ -1180,36 +1180,19 @@ public static function substr(string $string = null, int $start = 0, int $length
      * @param array $data Associative array with placeholders as
      *                    keys and replacements as values.
      *                    Supports query syntax.
-     * @param string|array|null $fallback An options array that contains:
-     *                                    - fallback: if a token does not have any matches
-     *                                    - callback: to be able to handle each matching result
-     *                                    - start: start placeholder
-     *                                    - end: end placeholder
-     *                                    A simple fallback string is supported for compatibility (but deprecated).
-     * @param string $start Placeholder start characters (deprecated)
-     * @param string $end Placeholder end characters (deprecated)
-     *
-     * @todo Remove `$start` and `$end` parameters, rename `$fallback` to `$options` and only support `array` type for `$options` in 3.7.0
-     *
+     * @param string|array|null $options An options array that contains:
+     *                                   - fallback: if a token does not have any matches
+     *                                   - callback: to be able to handle each matching result
+     *                                   - start: start placeholder
+     *                                   - end: end placeholder
      * @return string The filled-in string
      */
-    public static function template(string $string = null, array $data = [], $fallback = null, string $start = '{{', string $end = '}}'): string
+    public static function template(string $string = null, array $data = [], array $options = null): string
```

Comment:
Don't think it should be able to be `null`:

```suggestion
    public static function template(string $string = null, array $data = [], array $options = []): string
```